src/items/AppItem.tsx

```diff
@@ -88,12 +88,18 @@ const AppItem = ({
   const onLoad = (): void => setIsIFrameLoading(false);
 
   const appUrlWithQuery = useMemo(
-    () => appendQueryParamToUrl(appUrl, { itemId: item.id }),
+    () =>
+      appendQueryParamToUrl(appUrl, {
+        itemId: item.id,
+        // this ensures that the index.html can not be aggressively cached by the browser
+        ts: Date.now().toString(),
```

Comment:
Maybe `ts` is not explaining very well the purpose. What do you think about something like:

```suggestion
        timestampVersion: Date.now().toString(),
```

Reply:
@ReidyT Thank you for the comment, yes the param name is not very cleat, however its name does not really matter, only its presence and the fact that it changes often enough for the browser to not be able to cache the request is actually useful. See [this article](https://stackoverflow.com/questions/67219450/why-does-adding-a-random-get-parameter-make-favicon-show) on favicon. So maybe I could use `v` instead ? I think making it short is important too.